telegram/_inline/inlinekeyboardbutton.py

```diff
@@ -111,6 +112,10 @@ class InlineKeyboardButton(TelegramObject):
             in inline mode when they are currently in a private chat with it. Especially useful
             when combined with ``switch_pm*`` actions - in this case the user will be automatically
             returned to the chat they switched from, skipping the chat selection screen.
+
+            Tip:
+                This is similar to the new parameter :paramref:`switch_inline_query_chosen_chat`,
+                but for some reason, Telegram has not deprecated this parameter.
```

Comment:
But allows the user to select an arbitrary chat.

I see no need to comment in TGs choices if API design...

Reply:
Ok, how about this then: "This is similar to the new parameter :paramref:`switch_inline_query_chosen_chat`, but gives no control over which chats can be selected."

Need to distinguish between both these parameters which essentially do the same thing

Reply:
sounds good 👍